js/flightlog_fields_presenter.js

```diff
@@ -1125,6 +1158,32 @@ function FlightLogFieldPresenter() {
                         default:
                             return value.toFixed(0) + 'us';
                     }
+                case 'GPS_RESCUE_VELOCITY':
+                    switch (fieldName) {
+                        // debug 0 = Velocity P
+                        // debug 1 = Velocity I
+                        // debug 2 = Velocity D
+                        case 'debug[3]': // Velocity cm/s
+                            return (value / 100).toFixed(2) + 'm/s';
+                        default:
+                            return value.toFixed(0);
+                    }
+                case 'GPS_RESCUE_HEADING':
+                    switch (fieldName) {
+                        // debug 0 = error angle tenths of a degree
+                        case 'debug[0]': // error angle tenths of a degree
+                            return (value / 10).toFixed(1) + 'deg';
+                        // debug 1 = Velocity I
+                        case 'debug[1]': // angle to home in tenths of a degree
+                            return (value / 10).toFixed(1) + 'deg';
+                        // debug 2 = Velocity D
+                        case 'debug[2]': // commanded yaw rate in tenths of a degree
+                            return (value / 10).toFixed(1) + 'deg/s';
+                        case 'debug[3]': // roll angle in hundredths of a degree
+                            return (value / 100).toFixed(2) + 'deg';
+                        default:
+                            return value.toFixed(0);
+                    }
```

Comment:
The same, group cases, and some comments seems wrong from a copy&paste.
```suggestion
                case 'GPS_RESCUE_HEADING':
                    switch (fieldName) {
                        case 'debug[0]': // error angle tenths of a degree
                        case 'debug[1]': // angle to home in tenths of a degree                        
                            return (value / 10).toFixed(1) + 'deg';
                        case 'debug[2]': // commanded yaw rate in tenths of a degree
                            return (value / 10).toFixed(1) + 'deg/s';
                        case 'debug[3]': // roll angle in hundredths of a degree
                            return (value / 100).toFixed(2) + 'deg';
                        default:
                            return value.toFixed(0);
                    }
```

Reply:
thanks, updated